controller/embedding/manager.py

```diff
@@ -212,3 +213,17 @@ def __recreate_embedding(project_id: str, embedding_id: str) -> Embedding:
     connector.request_deleting_embedding(project_id, old_id)
     daemon.run(connector.request_embedding, project_id, new_embedding_item.id)
     return new_embedding_item
+
+
+def update_embedding_payload(
+    project_id: str, embedding_id: str, filter_attributes: List[str]
+) -> None:
+    notification.send_organization_update(
+        project_id=project_id,
+        message=f"upload_embedding_payload:{str(embedding_id)}",
+    )
```

Comment:
Usually we use notifications to send a "finished" note. here it's done before the payload has been updated. So if someone is listening to that value we will run in a racing condition. Sending it last instead should do the trick.

Reply:
My idea here was to send a notification when the update starts (so i can add the spinner on the front-end), because once this is finished we sent a notification 'embedding_updated' and the spinner is removed. Maybe a better naming or the approach is wrong?

Reply:
makes sense. maybe change it to `message=f"upload_embedding_payload:{str(embedding_id):start}"` to prevent irritation